admin/src/constants/variables.js

```diff
@@ -0,0 +1,15 @@
+export const production = import.meta.env.PROD;
+
+export const publicDir = () => {
+	if ( production ) {
+		return '/app/plugins/urlslab-plugin/admin/public'
+	}
+	return '';
+}
+
+export const domain = () => {
+	if ( production ) {
+		return ''
+	}
+	return 'http://liveagent.local';
```

Comment:
never like this pls ... domain should be computed from browser window url

Reply:
removed entirely – used for running in custom dev server outside of WP